search/utils.py

```diff
@@ -168,21 +169,35 @@ def search_index(tokens=None, repo_slug=None, sort_by=None, terms=None):
         search = search.query("match", repository=repo_slug)
     if sort_by is None:
         # Always sort by ID to preserve ordering.
-        search = search.sort("id")
+        search = search.sort(LoreSortingFields.BASE_SORTING_FIELD)
+    # Temporary workaround; the values in sorting.py should be updated,
+    # but for now Haystack is still using them. Also, the hyphen is
```

Comment:
Looking at all the occurrences of the term "haystack" (http://pastebin.com/DTVkGSk6) it appears that the use of the name was not replaced since the replacement had no impact on whether the code worked.  It's still confusing, though.  I'll create an issue proposing the replacement of the term "haystack" so the value of the change can be considered separately.